.release-please-manifest.json

```diff
@@ -1,3 +1,3 @@
 {
-  ".": "1.0.1-beta01"
-}
\ No newline at end of file
+  ".": "1.0.0"
```

Comment:
I think it makes more sense to make this `1.0.1`.

Reply:
Done